docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -13,6 +13,8 @@ You can push {es-sec} cases to these third-party systems:
 To push cases, you need to create a connector, which stores the information
 required to interact with an external system.
 
+NOTE: {kibana-ref}/pre-configured-connectors.html[Pre-configured connectors] cannot be used with cases. 
```

Comment:
Edit for consistency with usage in the Kibana docs (in https://github.com/elastic/kibana/pull/123876). On that Kibana PR, I also asked if the requirement applies to _any_ cases in Kibana (including Observability cases?) or just Security cases, which might be good to specify here too.
```suggestion
NOTE: {kibana-ref}/pre-configured-connectors.html[Preconfigured connectors] cannot be used with cases. 
```

Reply:
I considered specifying that this applied to cases in Security and Observability but thought it might appear misplaced and potentially give the impression that cases in these solutions were interchangeable somehow. That said, I'm all for challenging this instinct if I'm overthinking it.

Reply:
I think it's fine as is, now that we've confirmed that this refers to _all_ cases.